app/livechat/server/api/lib/visitors.js

```diff
@@ -62,13 +63,40 @@ export async function findChatHistory({ userId, roomId, visitorId, pagination: {
 	});
 
 	const total = await cursor.count();
-
 	const history = await cursor.toArray();
-
-	return {
-		history,
-		count: history.length,
-		offset,
-		total,
-	};
+	if(text == 'null'){
+		return {
+			history,
+			count: history.length,
+			offset,
+			total,
+			
+		};
+		
+	}else{
```

Comment:
This kind of code needs to be fixed. Please, add ESLint extension to your code editor.

```suggestion
	} else {
```

Reply:
done